### PR TITLE
React2 week1 meal-sharing

### PR DESCRIPTION
**Pull request by slastushevskyi**

### CODE REVIEW

File: src/client/components/MealListComponent.jsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React, { useEffect, useState } from "react";
-import MealListItem from "./MealListItem";
+import MealItem from "./MealItem";
```
Comment (LucasIversen) [inline src/client/components/MealListComponent.jsx:2]: Can't comment on code that has not been changed in this PR, but fetch seems good 👍

File: src/client/components/MealItem.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  return (
+    <>
+      <h2>{meal.title}</h2>
+      <p className="meal_desc">{meal.description}</p>
+      <p className="meal_price">{meal.price}</p>
+    </>
+  );
+};
+
+MealListItem.propTypes = {
```
Comment (LucasIversen) [inline src/client/components/MealItem.jsx:14]: Nice 👌

**Review by LucasIversen — APPROVED** [review]
Hi Nikita. Good job again. Feel free to merge it. If you want to expand on your meal sharing app, feel free to try out more of your REST calls on new routes :)

File: src/client/components/MealListComponent.jsx (hunk 1 above)
Comment (slastushevskyi) [reply src/client/components/MealListComponent.jsx:2]: Yeap, it works :) 
Thank you for reviews, btw!

**Pull request merged**